plugins/plugin-maven/che-plugin-maven-ide/src/main/java/org/eclipse/che/plugin/maven/client/wizard/MavenPagePresenter.java

```diff
@@ -212,11 +216,25 @@ public void go(AcceptsOneWidget container) {
       }
       updateDelegate.updateControls();
     }
+    projectService
+        .getItem(Path.valueOf(dataObject.getPath()).parent().append("pom.xml"))
+        .then(
+            result -> {
+              updateView(container, false);
```

Comment:
I think need to add some notification here, user must have information why create action disabled